distribution.ini

```diff
@@ -0,0 +1,8 @@
+[Global]
+id=mozilla-flatpak
```

Comment:
Do we need to change this given it's not in fact Mozilla's Flatpak if it comes from Endless? :)